.github/workflows/update-workflows-patch.yaml

```diff
@@ -0,0 +1,4 @@
+# By default, update-workflows only keeps itself up to date. This patch tells it to take control of additional workflows.
+- op: replace
+  path: /env/WORKFLOWS
+  value: update-workflows.yaml bump-version.yaml docker.yaml deploy.yaml typescript-ci.yaml
```

Comment:
This will try to pull in the `docker` and `deploy` workflows when it runs. I don't think we want those for this repo, do we?